Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/MainActivity.kt

```diff
@@ -108,6 +111,20 @@ class MainActivity : AppCompatActivity(), HasAndroidInjector {
             binding.mainBottomNavigation.updateCountBadge(R.id.covid_certificates_graph, count)
         }
 
+        vm.externalLinkEvents.observe(this) {
+            when (it) {
+                is HomeFragmentEvents.GoToCheckInsFragment -> navController.navigate(
```

Comment:
I would rename/create new set of Events for `MainActivity` in this case it is not part of `HomeFragment` itself 
Maybe `DeepLinkDirections` ? what do you think ?

Reply:
Yes, I wasn't sure wether to go this route or just throw them into an already existing Directions class. I will create a specific one just for them.